flink-libraries/flink-table/src/main/scala/org/apache/flink/table/expressions/ExpressionParser.scala

```diff
@@ -535,6 +536,13 @@ object ExpressionParser extends JavaTokenParsers with PackratParsers {
       case f ~ _ ~ _ => RowtimeAttribute(f)
     }
 
+  // key
```

Comment:
nit: this comment is not helpful it just duplicates the field name

Reply:
Ok, I will remove it.